pyfar/classes/transmission_matrix.py

```diff
@@ -0,0 +1,395 @@
+"""
+The following introduces the :py:class:`~TransmissionMatrix` class available in pyfar.
+
+Transmission matrices (short T-matrices) are a convenient representation of
+`Two-ports` (or `Quadrupoles`). These can represent systems from various fields,
+e.g. electrical circuits, mechanical vibration, (acoustic) transmission lines.
+
+Systems can be cascaded my multiplying consecutive T-matrices. Furthermore,
```

Comment:
Maybe add a short example here to highlight how easy it is to use?
```suggestion
Systems can be cascaded my multiplying consecutive T-matrices (e.g., by simply using the ``@`` operator).
```